crocket/src/main/java/com/crocket/view/DrawDirectionLine.java

```diff
@@ -8,39 +8,39 @@
 
 public class DrawDirectionLine extends JPanel {
     
-    private final int width = 100;
-    private final int height = 100;
-    private int startX = 50;
-    private int startY = 50;
+    private final int WIDTH = 100;
```

Comment:
Since these are assigned in the class body and presumably will be the same for all instances of the object, I would suggest making them public and static and remove the getter.

Reply:
fixed